js/page.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Created by Sych on 29.03.2016.
+ */
+"use strict";
+
+class Page {
+  constructor(options) {
+    this.DOM = options.DOM;
+
```

Comment:
I think name DOM can give some misunderstanding. Traditionally it called element, el, elem, etc. Of Couse you can use DOMElement or something like this.